CityPuzzle/Main Game/QuestPage.xaml.cs

```diff
@@ -127,6 +129,21 @@ private void Shuffle_Click(object sender, EventArgs e)
             shuffleButton.IsVisible = false;
         }
 
+        private void Map_Click(object sender, EventArgs e)
```

Comment:
Would be nice if you added a comment about the purpose of this function

Reply:
I added in quest page button show map and its function-"Map_Click"

Reply:
I meant in the code above the function. Something like 'displays map and a pin on current quest location'. It is usefull because 'Map_Click' doesnt say much about the purpose of the function

Reply:
Yes it doesn't provide much information- just that the button has a image "map" so I named it that way